app/api/answers/submit.feature

```diff
@@ -22,14 +22,18 @@ Feature: Submit a new answer
   Scenario: User is able to submit a new answer
     Given I am the user with ID "10"
     And time is frozen
+    And the following token "userTaskToken" signed by the app is distributed:
+      """
+      {
+        "idUser": "10",
+        "idItemLocal": "50",
+        "platformName": "{{app().TokenConfig.PlatformName}}"
+      }
+      """
     When I send a POST request to "/answers" with the following body:
       """
       {
-        "task_token": {{generateToken(map(
-          "idUser", "10",
-          "idItemLocal", "50",
-          "platformName", app().TokenConfig.PlatformName,
-        ), app().TokenConfig.PrivateKey)}},
+        "task_token": "{{userTaskToken}}",
```

Comment:
👍

Reply:
>The test I was mentioning would be a test where unmarshalling fails but still put something in the parsed value.

I'm not sure it is so although it can be so.